integration_tests/ci/sample.profiles.yml

```diff
@@ -33,7 +33,7 @@ integration_tests:
       role: "{{ env_var('CI_SNOWFLAKE_DBT_ROLE') }}"
       database: "{{ env_var('CI_SNOWFLAKE_DBT_DATABASE') }}"
       warehouse: "{{ env_var('CI_SNOWFLAKE_DBT_WAREHOUSE') }}"
-      schema: fivetran_log_integration_tests
+      schema: fivetran_log_integration_test
```

Comment:
curious why the switch?

Reply:
ah bc we needed to change the schema on spark?

Reply:
Oh weird... that may have been an accidentally deletion. Not sure why I would have removed this otherwise.

Reply:
yeah it looks like you added it after getting a `TableAlreadyExistsException` circleci error on spark so presuming it was a fix for that but applied across all WH profiles?